.github/workflows/pr-lint.yml

```diff
@@ -0,0 +1,22 @@
+name: Lint PR
+
+on:
+  pull_request_target:
+    types: [opened, reopened, edited]
+
+# This removes all unnecessary permissions, the ones needed will be set below.
+# https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token
+permissions: {}
+
+jobs:
+  lint-pr:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: xirzec/pr-lint@v1
```

Comment:
should we review the code for `xirzec/pr-lint` too?

Reply:
Happy to have folks take a look, not sure what the best way to do it would be but I'm open!